ices/82438.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
```

Comment:
Where did you take this example? Seems it's different from the snippet in the issue's OP (i.e. MCVE).

Reply:
Ah it was https://github.com/rust-lang/rust/issues/82526, nvm!